spec/defines/init_spec.rb

```diff
@@ -12,7 +12,6 @@
 
   context 'using minimum params' do
     it { is_expected.to create_user('rnelson0').with({
-      :comment          => 'Rob Nelson',
```

Comment:
Fixed. I got the rspec stuff backwards, by removing the expected check and not changing input parameters. Should be correct now.